### PR TITLE
reference numpy.org for all things numpy

### PR DESCRIPTION
**Pull request by mattip**
Part of numpy/numpy.org#25 and friends

### CODE REVIEW

File: www/getting-started.rst [hunk 1]
```diff
@@ -69,7 +69,7 @@ is to take a look at the following online resources:
 
 - `Python Scientific Lecture Notes <http://www.scipy-lectures.org/index.html>`_
 
-- `NumPy User Guide <http://docs.scipy.org/doc/numpy-dev/user/>`_ 
+- `NumPy User Guide <http://www.numpy.org/devdoc/user/>`_
```
Comment (rgommers) [inline www/getting-started.rst]: typo, should be `devdocs` not `devdoc`

File: www/docs.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@ Documentation
 
 Documentation for core SciPy Stack projects:
 
-* `Numpy <http://docs.scipy.org/doc/numpy/>`__
+* `Numpy <http://www.numpy.org>`__
```
Comment (rgommers) [inline www/docs.rst]: https://numpy.org/doc/

File: www/getting-started.rst (hunk 1 above)
Comment (mattip) [reply www/getting-started.rst]: fixed

File: www/docs.rst (hunk 2 above)
Comment (mattip) [reply www/docs.rst]: fixed

**Comment by mattip** [comment]
It seems #256 also would be good to review and merge as part of this change

**Comment by larsoner** [comment]
Comments addressed and LGTM, thanks @mattip

**Pull request merged**